pylintrc

```diff
@@ -272,7 +276,7 @@ ignore-mixin-members=yes
 
 # List of module names for which member attributes should not be checked
 # (useful for modules/projects where namespaces are manipulated during runtime
-# and thus existing member attributes cannot be deduced by static analysis
+# and thus existing member attributes cannot be deduced by static analysis)
```

Comment:
I think this is generated, so it needs to be fixed elsewhere too.

Reply:
I think it is generated from this file. I tested ``--generate-rc`` and it just copies the base `pylintrc` found in `./` to `stdout`.

Reply:
Wow, I wasn't careful enough with that file.

Reply:
Btw, that's also why `suggestion-mode` was missing I think. If it isn't added to `pylintrc` it doesn't get added anywhere.

Although, looking at the code I'm starting to doubt this... Anyway, might as well fix it here 😄 

https://github.com/PyCQA/pylint/blob/20304a6cfb0dedec8ca7878822b8db1fd3cc71cc/pylint/config/option_manager_mixin.py#L191-L224